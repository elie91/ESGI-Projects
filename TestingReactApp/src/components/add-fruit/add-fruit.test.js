import React from 'react';
import {shallow} from 'enzyme';

import AddFruit from './add-fruit.component';

describe('AddFruit component', () => {
  let wrapper;
  let mockMatch;
  let mockHistory;
  const mockRouteName = '/';
  let mockSubmitForm;

  beforeEach(() => {
    mockMatch = {
      path: '/add-fruit'
    };

    mockHistory = {
      push: jest.fn()
    };

    mockSubmitForm = jest.fn()

    const mockProps = {
      match: mockMatch,
      history: mockHistory,
      routeName: mockRouteName
    };

    wrapper = shallow(<AddFruit {...mockProps} />);
  });

  it('should render AddFruit component', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
