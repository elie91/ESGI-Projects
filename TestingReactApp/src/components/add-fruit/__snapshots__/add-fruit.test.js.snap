// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddFruit component should render AddFruit component 1`] = `ShallowWrapper {}`;
